source/vibe/core/connectionpool.d

```diff
@@ -125,6 +125,42 @@ final class ConnectionPool(Connection)
 		foreach (c; removed_conns)
 			disconnect_callback(c);
 	}
+
+	/** Removes an existing connection from the pool
```

Comment:
I think this needs to document the gotchas associated with using explicit `add`/`remove` calls, mainly telling when `remove` is allowed to be called (lock count zero or non-zero?) and potentially what this means when `add` is called with such a connection later on.

Reply:
Done